<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogClient</class>
 <widget class="QDialog" name="DialogClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLogin">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>client password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelAttacks">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>attacks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendDataBUtton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>send data</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sendDataLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="getDataButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>get data</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="getDataTextEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="attacksComboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="attacksTimeEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>time</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelProbability">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>probability</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addAttackButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>312</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="channelSpinBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelChannel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>channel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="attackButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>attack</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="attacksLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="probabilitySpinBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="channeledCheckBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>channeled</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="attacksTableWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="simulationCheckBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>simulation</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
